core/templates/dev/head/css/oppia.css

```diff
@@ -3111,14 +3111,15 @@ md-card.preview-conversation-skin-supplemental-card {
   font-weight: normal;
   hyphens: auto;
   line-height: 1.2em;
-  margin: 0.3em 0.5em;
+  margin: 0.3em;
   -ms-hyphens: auto;
   -moz-hyphens: auto;
   overflow-wrap: break-word;
-  padding: 0.2em;
+  padding: 0.3em;
   position: absolute;
   word-wrap: break-word;
   -webkit-hyphens: auto;
+  width: inherit;
```

Comment:
@anishbadri, can you please add this property in sorted order. (i.e, on line 3120)?

Reply:
Yup @DubeySandeep, I did make that change in the next commit